File: app/src/main/java/se/gustavkarlsson/aurora_notifier/android/background/providers/impl/OpenWeatherMapService.java
package se.gustavkarlsson.aurora_notifier.android.background.providers.impl;


import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

interface OpenWeatherMapService {

	@GET("weather")
	Call<OpenWeatherMapWeather> get(@Query("lat") double latitude, @Query("lon") double longitude, @Query("mode") String mode, @Query("appid") String appId);
}

File: app/src/main/kotlin/se/gustavkarlsson/skylight/android/services_impl/providers/openweathermap/OpenWeatherMapService.kt
package se.gustavkarlsson.skylight.android.services_impl.providers.openweathermap

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface OpenWeatherMapService {
    @GET("weather")
    fun get(@Query("lat") latitude: Double, @Query("lon") longitude: Double, @Query("mode") mode: String, @Query("appid") appId: String): Call<OpenWeatherMapWeather>
}
